spec/bike_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'bike'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.